cmd/logfile/logfile.go

```diff
@@ -27,9 +28,15 @@ func LoadParams(v *viper.Viper) (Params, error) {
 		debug = b
 	}
 
+	var sendDiagsOnErrors bool
+	if b := v.GetBool("settings.send_diagnostics_on_errors"); v.IsSet("settings.send_diagnostics_on_errors") {
+		sendDiagsOnErrors = b
+	}
+
 	params := Params{
-		ToStdout: v.GetBool("log-to-stdout"),
-		Verbose:  v.GetBool("verbose") || debug,
+		ToStdout:          v.GetBool("log-to-stdout"),
+		Verbose:           v.GetBool("verbose") || debug,
+		SendDiagsOnErrors: v.GetBool("send-diagnostics-on-errors") || sendDiagsOnErrors,
```

Comment:
Same for alphabetical order and we can replace it with `vipertools.FirstNonEmptyBool()` func, like:

```Go
SendDiagsOnErrors: vipertools.FirstNonEmptyBool("send-diagnostics-on-errors", "settings.send_diagnostics_on_errors")
```

Reply:
Should I do that for `debug` here too?

Reply:
Yes it makes sense to use vipertools instead.